docs/docs/setting-up/networking-instructions/networking.md

```diff
@@ -43,7 +47,7 @@ Bacalhau jobs are explicitly prevented from starting other Bacalhau jobs, even i
 
 ## Support for networked jobs on the public network
```

Comment:
Let's stop referring to this as the "public" network - we should be talking about the "demo" network, we don't want the impression that we're running a public network for everyone for free.

Reply:
done